vslib/src/SwitchStateBase.cpp

```diff
@@ -2612,3 +2616,16 @@ sai_status_t SwitchStateBase::set_voq_switch_attributes(
 
     return set(SAI_OBJECT_TYPE_SWITCH, m_switch_id, &attr);
 }
+
+sai_status_t SwitchStateBase::refresh_system_port_config_info(
+        _In_ const sai_attr_metadata_t *meta,
+        _In_ sai_object_id_t system_port_oid)
+{
+    SWSS_LOG_ENTER();
+
+    // Eventhough, this is not a read-only attribute, currently the system ports are configured
```

Comment:
SAI_SYSTEM_PORT_ATTR_CONFIG_INFO is create_only attribute, it should not be refreshed

Reply:
Yes. I'll remove this function

Reply:
Fixed